src/content/app/about/About.scss

```diff
@@ -18,18 +18,18 @@ $article-right-padding: 1.5rem;
   }
 }
 
+.textArticle {
+  height: calc(100vh - 370px);
```

Comment:
We shouldn't do this.

Reply:
I've added the height to the article area of the grid in `grid-template-rows` with the latest commit [4fef107](https://github.com/Ensembl/ensembl-client/pull/969/commits/4fef107518700c8b4f63d062511dca28c28df231). This should also fix the article height issue in the Help app.